frontend/src/components/dev/FeatureFlagsTable.tsx

```diff
@@ -53,6 +67,15 @@ export default function FeatureFlagsTable() {
           ))}
         </tbody>
       </Table>
+      <Button
+        data-testid={"reset-defaults"}
+        onClick={() => {
+          router.replace(pathname + "?" + createQueryString("_ff", "reset"));
```

Comment:
the `setQueryParam` function here should accomplish what we need, no need to create a new helper function as far as I can tell https://github.com/HHS/simpler-grants-gov/blob/c92704ac25dade39c6bee538ee73c70a0c89cdd9/frontend/src/hooks/useSearchParamUpdater.ts#L59